pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gtk-engine-murrine
+, gdk-pixbuf
+, librsvg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "2019-11-13";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "gtk-theme";
+    rev = "8ddf0fcad13c6a3f4fab2772b45561d74a8b4cce";
+    sha256 = "0byzxgd52291vsvwydafnq0xmnnggaryr9c9hfq5hxrd0rr08ibs";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    sassc
+    gtk3
+    inkscape
+    optipng
+  ];
+
+  buildInputs = [
+    gdk-pixbuf
+    librsvg
+  ];
+
+  propagatedUserEnvPkgs = [
+    gtk-engine-murrine
+  ];
+
+  postPatch = ''
+    for file in $(find -name render-\*.sh); do
+      patchShebangs $file
+
+      substituteInPlace "$file" \
+        --replace 'INKSCAPE="/usr/bin/inkscape"' \
+                  'INKSCAPE="inkscape"' \
+        --replace 'OPTIPNG="/usr/bin/optipng"' \
+                  'OPTIPNG="optipng"'
+    done
+  '';
+
+  meta = with stdenv.lib; {
+    description = "System76 Pop GTK+ Theme";
+    homepage    = "https://github.com/pop-os/gtk-theme";
+    license     = with licenses; [ gpl3 lgpl21 cc-by-sa-40 ];
+    platforms   = platforms.linux;
+    maintainers = with maintainers; [ elyhaka ];
```

Comment:
I think there's an entry in the code style conventions that suggest alignment, but it's not really encouraged and editors can just render text that way without edits.